test/helpers/helpers.go

```diff
@@ -373,3 +387,104 @@ func CheckCscSuccessfulDeletion(client test.FrameworkClient, cscName, namespace,
 
 	return nil
 }
+
+// RestartMarketplace scales the marketplace deployment down to zero and then scales
+// it back up to it's original number of replicas, and waits for a successful deployment.
+func RestartMarketplace(client test.FrameworkClient, namespace string) error {
+	marketplace := &apps.Deployment{}
+	err := client.Get(context.TODO(), types.NamespacedName{Name: "marketplace-operator", Namespace: namespace}, marketplace)
+	if err != nil {
+		return err
+	}
+	replicas := marketplace.Spec.Replicas
+	zero := int32(0)
+	marketplace.Spec.Replicas = &zero
+	err = client.Update(context.TODO(), marketplace)
+	if err != nil {
+		return err
+	}
+	err = client.Get(context.TODO(), types.NamespacedName{Name: "marketplace-operator", Namespace: namespace}, marketplace)
```

Comment:
I remember running into a problem where back to back updates to a deployment doesn't get picked up if the second update is just reverting to the original state. So you could cause a test flake here.

Reply:
I still think there is a risk of a test flake here if you are not waiting for scale down.